src/newgrf_roadstop.cpp

```diff
@@ -171,6 +171,22 @@ uint32 RoadStopScopeResolver::GetVariable(byte variable, uint32 parameter, bool
 			return ssl.grfid;
 		}
 
+		/* 16 bit road stop ID of nearby tiles */
+		case 0x6B: {
+			TileIndex nearby_tile = GetNearbyTile(parameter, this->tile);
+			uint32 grfid = this->st->roadstop_speclist[GetCustomRoadStopSpecIndex(this->tile)].grfid;
+
+			if (!IsRoadStopTile(nearby_tile)) return 0xFFFFFFFF;
+			if (!IsCustomRoadStopSpecIndex(nearby_tile)) return 0xFFFE;
+
+			const RoadStopSpecList ssl = BaseStation::GetByTile(nearby_tile)->roadstop_speclist[GetCustomRoadStopSpecIndex(nearby_tile)];
+			if (ssl.grfid != grfid) {
```

Comment:
Here it reads `ssl.grfid != grfid`, but in the station variant it reads `ssl.grfid == grfid`.

I know very little about NewGRFs, but it looks to me that one of the two isn't right. Couldn't tell you which of the two.

Reply:
This one is the incorrect one. The GRF local ID has no meaning to other GRFs.